insights/parsers/tests/test_numeric_user_group_name.py

```diff
@@ -0,0 +1,290 @@
+from insights.parsers.numeric_user_group_name import NumericUserGroupName
+from insights.tests import context_wrap
+
+ZERO = """
+/etc/passwd:0
+/etc/group:0
+""".strip()
+
+NONZERO_1 = """
+/etc/passwd:3
+/etc/group:0
+""".strip()
+
+NONZERO_2 = """
+/etc/passwd:0
+/etc/group:4
+""".strip()
+
+NONZERO_3 = """
+/etc/passwd:1
+/etc/group:1
+""".strip()
+
+UNSTRIPPED_ZERO = """
+
+/etc/passwd:0
+/etc/group:0
+
+"""
+UNSTRIPPED_NONZERO_1 = """
+
+/etc/passwd:3
+/etc/group:0
+
+"""
+UNSTRIPPED_NONZERO_2 = """
+
+/etc/passwd:0
+/etc/group:4
+
+"""
+UNSTRIPPED_NONZERO_3 = """
+
+/etc/passwd:1
+/etc/group:1
+
+"""
+
+# should produce no result
+ERRORS_1 = """
+
+/etc/passwd:1
+/bin/grep: /etc/group: No such file or directory
+
+"""
+
+ERRORS_2 = """
+
+/bin/grep: /etc/passwd: No such file or directory
+/etc/group:1
+
+"""
+
+ERRORS_3 = """
+
+/bin/grep: /etc/passwd: No such file or directory
+/etc/group:0
+
+"""
+
+# should produce no result
+ERRORS_4 = """
+
+/bin/grep: /etc/passwd: No such file or directory
+/bin/grep: /etc/group: No such file or directory
+
+"""
+
+# should produce no result
+ERRORS_5 = """
+
+/bin/grep: No such file or directory
+
+"""
+
+# Should produce a valid result because it contains valid lines.
+# The last valid lines are used.
+# /etc/passwd:1
+# /etc/group:1
+ERRORS_6 = """
```

Comment:
I'm really not convinced that we need a test of this kind of impossible output.  But whatever.

Reply:
You are right, I will fix it.